os/gl_window.h

```diff
@@ -15,8 +15,19 @@
 #define GL_UNSIGNED_SHORT_5_5_5_1 0x8034
 #endif
 
+#ifdef linux || _WIN32
```

Comment:
Do you think you could invert this ifdef chain so that macOS becomes the special case instead, i.e. (`ifdef __APPLE__` with the new code and the old Windows/Linux code in the else case)?

Reply:
Also note that it should be `__linux__` and not `linux`,

Reply:
Makes sense, fixed it